src/generation/parser.rs

```diff
@@ -3489,6 +3517,12 @@ fn gen_string_literal<'a>(node: &'a Str, context: &mut Context<'a>) -> PrintItem
       }
     };
 
+    fn is_valid_identifier(string_value: &String) -> bool {
+      if string_value.len() == 0 { false } else {
+        if string_value == "foo" { true } else { false }
```

Comment:
This is temp logic, pending my question about the best way to check for a valid JS identifier.